helm-chart/renku/values.yaml

```diff
@@ -870,20 +866,22 @@ dlf-chart:
     enabled: false
   dataset-operator-chart:
     enabled: true
+csi-rclone:
+  ## Name of the csi storage class to use for RClone/Cloudstorage. Should be unique per cluster.
+  storageClassName: csi-rclone 
+  csiNodepluginRclone:
+    nodeSelector: {}
+    tolerations: []
+    affinity: {}
 notebooks:
   cloudstorage:
-    ## If enabled this will allow mounting of buckets into user sessions
-    ## from any S3-compatible storage provider.
+    ## If enabled this will allow mounting of rclone cloudstorage
+    enabled: false
+    ## should match csi-rclone.storageClassName
+    storageClass: csi-rclone
+    installRCloneCSI: false
```

Comment:
Now since we have a single helm chart we do not have to repeat stuff in the values file between here and the top level `csi-rclone` section. Lets just have a single one at `csi-rclone` and have the notebook service templates pull whatever they need from there. Controlling whether the csi rclone helm chart is installed should be controlled from the top level `csi-rclone`  section. Also you should error out the helm install if `notebooks.cloudstorage` is enabled but the helm chart for csi-rclone is not installed.